brevbaker/src/testFixtures/kotlin/no/nav/brev/brevbaker/IntegrationTestUtils.kt

```diff
@@ -0,0 +1,219 @@
+package no.nav.brev.brevbaker
```

Comment:
Denne fila er rein kopi av den som ligg i pensjon-brevbaker. Tanken er å, når alt er flytta over for både etterlatte og pesys, slette frå pensjon-brevbaker og bruke denne derifrå også.